src/python/grpcio_tests/tests/fork/client.py

```diff
@@ -63,12 +63,12 @@ def _test_case_from_arg(test_case_arg):
 
 def test_fork():
     logging.basicConfig(level=logging.INFO)
-    args = _args()
-    if args.test_case == "all":
+    args = vars(_args())
```

Comment:
What was the intent of this change?

Edit: I see. Unified interface for both CLI and programmatic invocation. :+1: